package.json

```diff
@@ -46,7 +41,12 @@
     "test": "jest",
     "test:watch": "npm run test -- --watch",
     "test:coverage": "npm run test -- --coverage",
-    "open:coverage": "npm run test:coverage && opn ./coverage/lcov-report/index.html"
+    "open:coverage":
+      "npm run test:coverage && opn ./coverage/lcov-report/index.html",
+    "build:server": "rimraf ./dist &&  babel api --out-dir ./dist/api",
+    "build:client": "webpack -p --config webpack.prod.config.js --progress",
```

Comment:
Hmm not sure about this one. Maybe development should be the default? What do you all think?

Reply:
For which script? build:client?

Reply:
You can disregard my previous comment. I forgot that for development there are other tasks, so it's ok to have build executing the PROD webpack version.